File: src/gestorAplicacion/clasesPrincipales/Conductor.java
package gestorAplicacion.clasesPrincipales;

import java.io.Serializable;
import java.util.ArrayList;

public class Conductor implements Serializable {
	
	private static final long serialVersionUID = 2L;
	
	private String nombre;
	private long cedula;
	private long celular;
	private int sueldo;
	private boolean disponible;
	private static ArrayList<Conductor> conductores = new ArrayList<>();
	
	public Conductor(String nombre, long cedula, long celular, int sueldo) {
		this.nombre = nombre;
		this.cedula = cedula;
		this.celular = celular;
		this.sueldo = sueldo;
		setDisponible(true);
        Conductor.conductores.add(this);
	}

	//GETs y SETs
	public String getNombre() {
		return nombre;
	}
	
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	
	public long getCedula() {
		return cedula;
	}
	
	public void setCedula(int cedula) {
		this.cedula = cedula;
	} 
	
	
	public long getCelular() {
		return celular;
	}

	public void setCelular(int celular) {
		this.celular = celular;
	}

	public int getSueldo() {
		return sueldo;
	}

	public void setSueldo(int sueldo) {
		this.sueldo = sueldo;
	}

	public static ArrayList<Conductor> getConductores() {
		return conductores;
	}

	public static void setConductores(ArrayList<Conductor> conductores) {
		Conductor.conductores = conductores;
	}

	public boolean isDisponible() {
		return disponible;
	}

	public void setDisponible(boolean disponible) {
		this.disponible = disponible;
	}

	//Metodos
	
}

File: Python/gestor_aplicacion/clasesPrincipales/Conductor.py
class Conductor:
    _conductores = []

    def __init__(self, nombre, cedula, celular, sueldo):
        #instance fields found by Java to Python Converter:
        self._nombre = nombre
        self._cedula = cedula
        self._celular = celular
        self._sueldo = sueldo
        Conductor._conductores.append(self)

    #GETs y SETs
    def getNombre(self):
        return self._nombre

    def setNombre(self, nombre):
        self._nombre = nombre

    def getCedula(self):
        return self._cedula

    def setCedula(self, cedula):
        self._cedula = cedula


    def getCelular(self):
        return self._celular

    def setCelular(self, celular):
        self._celular = celular

    def getSueldo(self):
        return self._sueldo

    def setSueldo(self, sueldo):
        self._sueldo = sueldo

    @classmethod
    def getConductores(cls):
        return Conductor._conductores

    @classmethod
    def setConductores(cls, conductores):
        Conductor._conductores = conductores
